earthpy/plot.py

```diff
@@ -389,7 +390,7 @@ def plot_rgb(
         ...     img_array = src.read()
         >>> # Ensure the input array doesn't have nodata values like -9999
         >>> ep.plot_rgb(img_array)
-        <matplotlib.axes._subplots.AxesSubplot object at 0x...
+        <AxesSubplot:>
```

Comment:
oh is this it? the return is just slightly different now? maybe they changed the API a bit?

Reply:
Ya! Sorry I didn't document this well, there were two instances where docs was failing because the output from matplotlib changed slightly and the expected output didn't match what was given.